core/templates/dev/head/dashboard/dashboard.html

```diff
@@ -104,6 +111,12 @@ <h2 style="margin-bottom: -20px; margin-left: 52px;">Creator Dashboard</h2>
                       Private
                     </span>
                   </span>
+                  <div ng-if="exploration.num_open_threads == 0 && exploration.status !== 'private'">
```

Comment:
Just wondering, why would you only allow sharing if the number of open feedback threads is zero?